lib/form/deposit.js

```diff
@@ -1,4 +1,4 @@
-import Configs from '@tokoinofficial/t-chain-payment/lib/config';
+import Configs from '../config';
```

Comment:
Must use `window._tpayment.configs` (switchable between sandbox and production mode) as `Configs` is production mode only.